client/driver/docker_coordinator.go

```diff
@@ -98,6 +104,12 @@ type dockerCoordinator struct {
 	// only have one request be sent to Docker
 	pullFutures map[string]*pullFuture
 
+	// pullLoggers is used to track the LogEventFn for each alloc pulling an image
```

Comment:
Worth adding a comment here about why it needs a slice of pull loggers if the map is keyed by docker image id.